scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -20,18 +21,43 @@ case class ExplicitReturnTypes(mirror: Mirror)
 
   def defnName(defn: Defn): Option[Name] = Option(defn).collect {
     case Defn.Val(_, Seq(Pat.Var.Term(name)), _, _) => name
+    case Defn.Var(_, Seq(Pat.Var.Term(name)), _, _) => name
     case Defn.Def(_, name, _, _, _, _) => name
   }
 
+  def visibility(mods: Traversable[Mod]): MemberVisibility =
+    mods
+      .collectFirst {
+        case _: Mod.Private => MemberVisibility.Private
+        case _: Mod.Protected => MemberVisibility.Protected
+      }
+      .getOrElse(MemberVisibility.Public)
+
+  def kind(defn: Defn): Option[MemberKind] = Option(defn).collect {
+    case _: Defn.Val => MemberKind.Val
+    case _: Defn.Def => MemberKind.Def
+    case _: Defn.Var => MemberKind.Var
+  }
+
   def parseDenotationInfo(denot: Denotation): Option[Type] = {
     val base =
-      if (denot.isVal) denot.info
+      if (denot.isVal || denot.isVar) denot.info
       else if (denot.isDef) denot.info.replaceFirst(".*\\)", "")
       else {
         throw new UnsupportedOperationException(
           s"Can't parse type for denotation $denot, denot.info=${denot.info}")
       }
-    base.parse[Type].toOption
+    if (denot.isVal || denot.isDef)
+      base.parse[Type].toOption
+    else
+      /*
+      Currently a symbol of Var points to its setter function.
+      That's why its argument type should be extracted via pattern-match.
```

Comment:
Let's point this to a getter. This would be a simple matter of changing https://github.com/scalameta/scalameta/blob/48eff97ac6951e5b58eb2ec7251f69d53a38a73a/scalahost/nsc/src/main/scala/scala/meta/internal/semantic/DenotationOps.scala#L15.

Would you consider contributing this to scalameta/scalameta?